### PR TITLE
Add scroll to bottom button in chat page

### PR DESCRIPTION
**Pull request by jamessspanggg**
- Scroll to bottom button will only show when user is not at the bottom of the page.
- Number of unread messages will be shown in the scroll to bottom button, once scroll to the bottom, the unread messages count will be reset and the scroll to bottom button won't be shown

### CODE REVIEW

File: src/components/chat/modules/ChatDialogMessages.js [hunk 1]
```diff
@@ -228,6 +286,24 @@ const ChatDialogMessages = ({
...
       </MessageContainer>
+      {isShowScrollerButton && (
+        <ScrollToBottomContainer>
+          <ScrollerButtonContainer>
+            <Button
+              circled
+              transparent
+              iconLeft={<ChevronDown />}
+              asComponent={ScrollToBottomButton}
+              onClick={() => bottomOfScrollerRef.current.scrollIntoView()}
```
Comment (kohchihao) [inline src/components/chat/modules/ChatDialogMessages.js:297]: https://developer.mozilla.org/en-US/docs/Web/API/Element/scrollIntoView

Consider putting the `{ behavior: "smooth" }`

**Review by kohchihao — COMMENTED** [review]
Look goods except maybe need some animations

File: src/components/chat/modules/ChatDialogMessages.js (hunk 1 above)
Comment (jamessspanggg) [reply src/components/chat/modules/ChatDialogMessages.js]: yep done!

**Review by kohchihao — APPROVED** [review]
LGTM 👍

**Pull request merged**